engine/xml-config/src/main/java/org/datacleaner/configuration/JaxbConfigurationReader.java

```diff
@@ -891,7 +892,7 @@ private Datastore createDatastore(String name, ElasticSearchDatastoreType datast
             }
         }
 
-        return new ElasticSearchDatastore(name, ClientType.TRANSPORT, hostname, port, clusterName, indexName, tableDefs,
+        return new ElasticSearchDatastore(name, ClientType.valueOf(clientType), hostname, port, clusterName, indexName, tableDefs,
```

Comment:
Ouch, looks like TRANSPORT was hardcoded before?! That's a bug that I guess we also fix now then. Or am I missing something?